### DISCUSSION TITLE
How do i disable devtools (vue) on vuepress-next

### DISCUSSION THREAD

**Discussion by devPaulRS** [vuepress/core#1146] [Q&A]
I would like to disable the devTools, I've tried the following:

```
Vue.config.devtools = false
Vue.config.debug = false
Vue.config.silent = true
```

To no success. Any ideas besides using production?

**Comment by meteorlxy** [comment]
It's disabled in production mode by default. You means to disable it in dev mode? 🤔

**Reply by devPaulRS** [reply]
@meteorlxy Yes to disable it in dev mode, I want to disable multiple things in developer mode and make a control api for enable/disable dev mode with authentication.

**Reply by devPaulRS** [reply]
Without using the build (static) since I'm using vite for hot loading.

**Reply by Mister-Hope** [reply]
For these kind of unusual usage, I think that you may need to patch and edit the plugin yourself.
